src/protocol.rs

```diff
@@ -310,32 +318,26 @@ impl ParticipantConfig {
     /// [`ParticipantIdentifier`]s.
     ///
     /// This method implies the existence of a trusted third party that
-    /// generates the IDs and gives them to participants.
+    /// generates the IDs; that's why it's only available for testing right
+    /// now.
```

Comment:
todo: remove last clause ("that's why it's only available...").

In #302, Omar made this method public to use in the example.

Reply:
nit: Sorry, I might not have been clear. The last line of documentation here should just read:
```
/// This method implies the existence of a trusted third party that
/// generates the IDs.
```